src/components/AppTodoContext/AppTodoContext.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, useMemo, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { Props, Value } from './AppTodoContext.types';
+import { ErrorType } from '../Error/Error.types';
+
+export const AppTodoContext = React.createContext<Value>({
+  todos: [],
+  setTodos: () => {},
+  visibleTodos: [],
+  setVisibleTodos: () => {},
+  tempTodo: null,
+  setTempTodo: () => {},
+  processingTodoIDs: [],
+  setProcessingTodoIDs: () => {},
+  completedTodos: [],
+  uncompletedTodos: [],
+  errorMessage: 'No error' as ErrorType,
```

Comment:
The default value for `errorMessage` should be `ErrorType.NoError` instead of the string 'No error'.